DeliveryServiceModel/LinqExamples.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Linq;
+namespace DeliveryServiceModel
+{
+    public class LinqExamples
+    {
+        private readonly TestData data;
+
+        public LinqExamples(TestData sourceData)
+        {
+            data = sourceData;
+            data.FillData();
+        }
+
+        public void QueryTask()
+        {
+            //Task 1
+            Console.WriteLine("Task 1");
+            var stockCollection = data.suppliers.SuppliersCollection.SelectMany(x => x.Stock);
```

Comment:
please create separate method for each task (only for logic)
UI things such as Console.WriteLine would not be there